src/SwaggerProvider.DesignTime/OperationCompiler.fs

```diff
@@ -11,6 +11,12 @@ open FSharp.Data
 open Microsoft.FSharp.Quotations
 open Microsoft.FSharp.Quotations.ExprShape
 open System.Text.RegularExpressions
+open System.IO
+
+type BodyType =
```

Comment:
modeling the body content like this made it easier to switch between formdata and multipart form data when we get parameters lists that have normal form data and files